.stylelintrc.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
```

Comment:
Same as for `eslint`: Why not use `.stylelintrc` and it should be move on its own repository (i.e. `stylelint-config-exolnet`)

Reply:
We deceided to go with a `.stylelintrc.json` file since the text editor can better interpret the file, and help with syntax checking and highlighting.